JASP-Engine/JASP/R/MixedModelsCommon.R

```diff
@@ -2918,30 +2877,48 @@
     
     for (cf1 in 1:coefs1) {
       for (cf2 in 1:coefs2) {
+
+        coefs_trend <- attr(samples1, "estimate")
+        coefs_trend <- gsub("trend ('", "", coefs_trend, fixed = TRUE)
+        coefs_trend <- gsub("')", "", coefs_trend, fixed = TRUE)
+        coefs_trend <- strsplit(coefs_trend, ",")
+        
         coefs1_name <-
           paste(.unv(unlist(
             strsplit(dimnames(samples1)$Parameter[cf1], ",")
           )), collapse = ":")
         coefs1_name <- gsub(" ", "", coefs1_name, fixed = TRUE)
-        coefs1_name <-
-          .mmVariableNames(coefs1_name, options$fixedVariables)
+        coefs1_name <- .mmVariableNames(coefs1_name, options$fixedVariables)
         coefs2_name <-
           paste(.unv(unlist(
             strsplit(dimnames(samples2)$Parameter[cf2], ",")
           )), collapse = ":")
         coefs2_name <- gsub(" ", "", coefs2_name, fixed = TRUE)
-        coefs2_name <-
-          .mmVariableNames(coefs2_name, options$fixedVariables)
+        coefs2_name <- .mmVariableNames(coefs2_name, options$fixedVariables)
         
+        if(length(coefs_trend) > 0){
+          for(cft in coefs_trend){
+            if(cft %in% strsplit(pars[[1]], ":")[[1]] && !grepl(.unv(cft), coefs1_name)){
+              coefs1_name <- paste0(coefs1_name, ":", .unv(cft))
+            }
+          }
+        }
+        if(length(coefs_trend) > 0){
+          for(cft in coefs_trend){
+            if(cft %in% strsplit(pars[[2]], ":")[[1]] && !grepl(.unv(cft), coefs2_name)){
+              coefs2_name <- paste0(coefs2_name, ":", .unv(cft))
+            }
+          }
+        }
```

Comment:
shouldn't this be a function? The exact same code is also on lines 2846-2852, and here twice.

Reply:
yes, true. It is supposed to be a temporary solution - it's fixing a problem in `stanova` - we should get rid of this code with the next release. Changed